README.md

```diff
@@ -10,6 +10,49 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+What does OpenCraft Instance Manager do and how to use it
+---------------------------------------------------------
+
+This section focuses on how to use the web interface, as opposed to how to install the code,
+debug it or develop it (see next sections for that).
+
+Ocim is a web interface to manage a list of Open edX servers. With some clicks you can create
+new servers, check their status and their configuration.
+It looks like this:
+
+![Ocim's main screen](documentation/ocim_main_screen.png)
+
+At the left there's a list of instances, and each instance has many appservers.
+We can create a new appserver through the **Launch new AppServer** button;
+it will automatically get the current configuration from the instance settings
+and use it for this server.
+After 1 to 2 hours, it will finish and then you need to **activate** the new one
+and **deactivate** the old one, to make the domain name direct to the new one.
+Normally we want just 1 active appserver per instance.
+Before activating a server, there's the option to test it through a password-protected link.
+
+Sometimes Open edX playbook fails, and then you need to read the log,
+which is shown in real-time in the web.
+You can fix the settings and then spawn another server.
+Failed servers are automatically cleaned up after some days.
+An important feature is that Ocim *grants SSH access to members of our organization* (OpenCraft)
+so you can always SSH to an appserver's IP, *even if Open edX's deployment failed*, and then debug it.
+You can use your GitHub username and key.
+
+To create a new instance, you use Django's admin and you need to fill in the domain name,
+the prefixed domain names (for Studio, e-commerce, etc.), the edx-platform/configuration branches to use,
+and extra ansible variables to pass to Open edX's playbook (if any).
+The instance settings are used for new deployments only
+(changing the instance settings doesn't retroactively redeploy appservers).
+
+How do we use Ocim at OpenCraft, as of April 2018:
```

Comment:
I don't know if this belongs in the public documentation or not. It may be helpful to others in the community wanting to use Ocim, but it doesn't have to be specific about how we use it, otherwise it may give away some undesirable information and/or have to be updated in case we use it differently later (evidenced by the fact that we had to enter a date).

But I do get the point here, and it's definitely good to have -- so maybe instead just re-word it to be more generic, i.e. "Some use cases of Ocim:", going on to explain what special things one can do by adopting Ocim. Basically, "sell it" to them in this paragraph. This paragraph may also want to go up a bit higher, in the main header as its own section, by updating/replacing what's currently there.

Reply:
I tried to write the things that I was missing when I started: Ocim's URL (I didn't realize at first that „Console“ and „Ocim“ were the same), the stage Ocim, ephemeral DBs, …
But ok, if it can't go here, then better leave it for the onboarding course, or mentoring.
The use cases are already explained in the introduction, and the note about ephemeral DBs is too specific, so better I remove all this.